docs/release-notes/1.4.0-preview/release-1.4.0-preview.md

```diff
@@ -0,0 +1,88 @@
+# [ML.NET](http://dot.net/ml) 1.4.0-preview
+
+## **New Features**
+- **Deep Neural Networks Training (0.16.0-preview)** ([#4151](https://github.com/dotnet/machinelearning/pull/4151))  
+
+  Improves the in-preview `ImageClassification` API further:
+  - Increases DNN training speed by ~10x.
```

Comment:
You need to say "compared to what" because when using other approaches it might be faster/slower.
So it should say "compared to the same API in 0.15.1 release".